src/generator/AutoRest.Go/TemplateModels/VersionTemplateModel.cs

```diff
@@ -15,12 +15,14 @@ public class VersionTemplateModel : ServiceClient
         public static readonly List<string> StandardImports = new List<string> { "fmt" };
 
         public readonly string PackageName;
+        public readonly string[] VersionString;
 
-        public VersionTemplateModel(ServiceClient serviceClient, string packageName)
+        public VersionTemplateModel(ServiceClient serviceClient, string packageName, string[] versionString)
```

Comment:
minor: I'd just name `versionString` to `version` as IMO `versionString` misleading name for string array.